docs/operations/upgrade.md

```diff
@@ -6,6 +6,16 @@ Unfortunately Loki is software and software is hard and sometimes things are not
 
 On this page we will document any upgrade issues/gotchas/considerations we are aware of.
 
+## 1.5.0
```

Comment:
I'm torn - Cortex included many breaking changes in it's _major_ (1.0) release. Should we increment to 2.0 here?